web_src/js/features/tableSort.js

```diff
@@ -0,0 +1,68 @@
+export default function initTableSort() {
+  $('th[data-sortt-asc]').each(function () {
+    // get data
+    const { sorttAsc, sorttDesc, sorttDefault } = $(this)[0].dataset;
+
+    // add onclick event
+    $(this).on('click', () => {
+      tableSort(sorttAsc, sorttDesc, sorttDefault);
+    });
+
+    // add arrow to column
+    const arrow = getArrow(sorttAsc, sorttDesc, sorttDefault);
+    // if function got a match ...
+    if (arrow !== false) {
+      $(this).prepend(`${arrow} `);
+    }
+  });
+}
+
+function tableSort(normsort, revsort, isdefault) {
+  if (!(normsort)) return false;
+  if (!(revsort)) revsort = '';
+
+  const url = new URL(window.location);
+  let url_sort = url.searchParams.get('sort');
+  if ((url_sort === null) && isdefault) url_sort = normsort;
+
+
+  if (url_sort !== normsort) {
+    url.searchParams.set('sort', normsort);
+  } else if (revsort !== '') {
+    url.searchParams.set('sort', revsort);
+  }
+
+  window.location.replace(url.href);
+}
+
+// create global function with main routine
+function getArrow(normsort, revsort, isdefault) {
+  // arrows
+  const arrow_down = '⯆'; // U+2BC6
+  const arrow_up = '⯅'; // U+2BC5
```

Comment:
Can move those constants out of the function onto top-scope (preferable on top of the file).

Reply:
I'll do & test tomotow - thanks!